CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+- Fixes an issue where immutable params were not sent during ext:configure.
+- Includes latest features and improvements from production in the Firestore Emulator.
+- Fixes issue where all database functions triggered on the default namespace (#2501)
+- Fixes issue where rules paths were not normalized before reading (#2544)
+- Replace `tslint` with `eslint` as the default linter in new Cloud Functions for Firebase directories. (#1663)
```

Comment:
This changelog needs to be fixed. I updated the branch via the GH UI so you may have to `git pull` on your end (or edit in the UI). The changelog diff should be just this ts->es change.

Reply:
@bkendall Thank you for your help. I removed the others.